docs/features/files.md

```diff
@@ -11,10 +30,14 @@ new GenericContainer(...)
 
 ## Volume mapping
 
+!!! warning
+    This feature is not supported for all Docker environments. Consider using the `copy` APIs instead.
```

Comment:
we need to make sure that this does not read as Testcontainers' limitation, but environments'. Maybe mention an example where test host's file system won't match Docker's.